options_compile.cpp

```diff
@@ -263,6 +276,17 @@ std::string EffectiveOptionsFilter::processOptions(const OpenCLArgList &args,
       {"cl_intel_subgroups", true},
       {"cl_intel_subgroups_short", true}};
 
+#ifdef PCH_EXTENSION
```

Comment:
I think a new variable should be defined in CMakeLists.txt
set(PCH_EXTENSION "" CACHE STRING
if (PCH_EXTENSION not empty)
  add_defitions(-DPCH_EXTENSION=${PCH_EXTENSION})

Reply:
@hewj03, thanks for your suggestions, I have updated the patch.

Reply:
it would be better to write like this so that there is no need to clear the map.
#ifdef PCH_EXTENSION
  std::map<std::string, bool> extMap;
  .. fill the map ...
#else
  std::map<std::string, bool> extMap{
      {"cl_khr_3d_image_writes", true},
        ...
#endif

Reply:
done